ale/drivers/lo_drivers.py

```diff
@@ -218,4 +219,173 @@ def naif_keywords(self):
                 f"INS{self.ikid}_ITRANSS": itranss,
                 f"INS{self.ikid}_ITRANSL": itransl}
 
-        return self._naif_keywords
\ No newline at end of file
+        return self._naif_keywords
+
+
+class LoMediumCameraIsisLabelNaifSpiceDriver(Framer, IsisLabel, NaifSpice, NoDistortion, Driver):
+
+    @property
+    def lo_detector_map(self):
+        return {'Lunar Orbiter 1': {'name':'LO1_MEDIUM_RESOLUTION_CAMERA', 'id':-531002},
+                'Lunar Orbiter 2': {'name':'LO2_MEDIUM_RESOLUTION_CAMERA', 'id':-532002},
+                'Lunar Orbiter 3': {'name':'LO3_MEDIUM_RESOLUTION_CAMERA', 'id':-533002},
+                'Lunar Orbiter 4': {'name':'LO4_MEDIUM_RESOLUTION_CAMERA', 'id':-534002},
+                'Lunar Orbiter 5': {'name':'LO5_MEDIUM_RESOLUTION_CAMERA', 'id':-535002}}
+
+    @property
+    def instrument_id(self):
+        """
+        Returns the ID of the instrument.
+
+        Returns
+        -------
+        : str
+          Name of the instrument
+        """
+        lookup_table = {'Medium Resolution Camera': self.lo_detector_map[self.spacecraft_name]['name']}
+        return lookup_table[super().instrument_id]
+
+    @property
+    def ikid(self):
+        """
+        Returns the Naif ID code for the instrument
+        Expects the instrument_id to be defined. This must be a string containing
```

Comment:
Can we update this doc string to accurately represent what needs to be defined here?